pkg/api/admin/service/namespace/validator.go

```diff
@@ -0,0 +1,20 @@
+package namespace
+
+import (
+	"regexp"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api"
+)
+
+const namespaceValidationMessage = "namespace code is invalid -- must be 2-12 letters, numbers, dash, or underscore"
+
+// validation rule for namespace code
+var validNamespaceCode = regexp.MustCompile(`^[\w\d-_]{2,12}$`)
```

Comment:
@jgiannuzzi do we have specific validation rule for namespace names? I just made a guess here